src/gluonnlp/model/sampled_block.py

```diff
@@ -91,7 +91,7 @@ def hybrid_forward(self, F, x, sampled_values, label, w_all, b_all):
             label_vec = F.reshape(label, (-1, 1))
             new_label_sampled = F.zeros_like(pred_sampled)
             new_label_true = F.ones_like(label_vec)
-            new_label = F.Concat(new_label_sampled, new_label_true, dim=1)
+            new_label = F.Concat(new_label_true, new_label_sampled, dim=1)
```

Comment:
What models does this change affect?

Reply:
language model trained with NCE. gluonnlp doesn't provide any script using NCE.